docs/source/release/v3.11.0/ui.rst

```diff
@@ -58,6 +58,7 @@ SliceViewer Improvements
 VSI Improvments
 ---------------
 - ParaView has been updated to to `v5.4.0 <https://blog.kitware.com/paraview-5-4-0-release-notes/>`_.
+- Multislice view uses a custom representation to speed up slicing by taking advantage of the consistent bin sizes in a MDHistoWorkspace.
```

Comment:
A link to what a representation is and a second sentence that explains what this actually means to the user would help.